### PR TITLE
README.md fixes

### PR DESCRIPTION
**Pull request by SevenSpheres**
I made some small fixes in the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,10 +20,10 @@ This program is distributed in the hope that it will be useful, but WITHOUT
...
 FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details,
 which you should have received along with this program (filename: COPYING).
-If not, request a copy from:
-Free Software Foundation, Inc.
-59 Temple Place - Suite 330
-Boston, MA  02111-1307
+If not, request a copy from:  
+Free Software Foundation, Inc.  
+59 Temple Place - Suite 330  
+Boston, MA  02111-1307
```
Comment (375gnu) [inline README.md]: please remove these extra spaces

File: README.md [hunk 2]
```diff
@@ -62,8 +62,8 @@ button, and you're off.
 
 That covers the very basics.  For a more in-depth look at Celestia and the
 controls available to you, download the "Celestia User's Guide" (written by 
-Frank Gregorio), available in several languages, from:
-  https://celestia.space/guides.html
+Frank Gregorio), available in several languages, from:  
+  https://celestia.space/guides.html
```
Comment (375gnu) [inline README.md]: and here to

File: README.md [hunk 3]
```diff
@@ -117,20 +117,21 @@ types of catalog files:
...
-| Chris Laurel, Clint Weisbrod, Fridger Schrempp, Bob Ippolito, Christophe Teyssier, Hank Ramsey, Grant Hutchison, Pat Suwalski, Toti , Da Woon Jung, Vincent Giangiulio, Andrew Tribick | Deon Ramsey, Christopher ANDRE, Colin Walters, Peter Chapman, James Holmes, Harald Schmidt, Sergey Leonov, Alexell, Dmitry Brant, Janus, Pirogronian, Hleb Valoshka | Frank Gregorio, Hitoshi Suzuki, Christophe Teyssier, Diego Rodriguez, Don Goyette, Harald Schmidt | Creators of scientific data base, texture maps, 3D models and used librarys, you can see in full README.|
+| Chris Laurel, Clint Weisbrod, Fridger Schrempp, Bob Ippolito, Christophe Teyssier, Hank Ramsey, Grant Hutchison, Pat Suwalski, Toti, Da Woon Jung, Vincent Giangiulio, Andrew Tribick, Hleb Valoshka, Łukasz Buczyński | Deon Ramsey, Christopher Andre, Colin Walters, Peter Chapman, James Holmes, Harald Schmidt, Nils Larsson, Sergey Leonov, Alexell, Dmitry Brant, Janus, Li Linfeng | Selden Ball, Frank Gregorio, Hitoshi Suzuki, Christophe Teyssier, Diego Rodriguez, Don Goyette, Harald Schmidt | Creators of scientific database, texture maps, 3D models and used libraries, you can see in full README.|
```
Comment (375gnu) [inline README.md]: please move Li Linfeng to authors

**Comment by SevenSpheres** [comment]
> and please merge all your commits into a single one

How do I do that?

> please remove these extra spaces
> and here to

I added those spaces so there would be line breaks there. Should I replace them with `<br>` tags?

> please move Li Linfeng to authors

Will do.

**Comment by 375gnu** [comment]
> How do I do that?

`git rebase -i HEAD~4` the replace all but the first `pick` with `f`

> I added those spaces so there would be line breaks there. Should I replace them with `<br>` tags?

Yes, please.

**Comment by SevenSpheres** [comment]
> git rebase -i HEAD~4 the replace all but the first pick with f

Okay I'm clueless - do I need to install some git client?

**Comment by 375gnu** [comment]
> > git rebase -i HEAD~4 the replace all but the first pick with f
> 
> Okay I'm clueless - do I need to install some git client?

Yes, a local git client is required.

**Comment by SevenSpheres** [comment]
> git rebase -i HEAD~4 the replace all but the first pick with f

Did the above and then this:
`git push origin master --force`
I see no changes.

**Comment by SevenSpheres** [comment]
I tried to merge the commits another way and now the pull request is closed...

**Comment by pirogronian** [comment]
@SevenSpheres You closed it personally :-) Probably by accident. But it can be easly reversed.

**Comment by 375gnu** [comment]
@pirogronian it was closed because of #458

**Pull request closed without merging**